exporter/logicmonitorexporter/logs_exporter_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logicmonitorexporter
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/zap"
+)
+
+func Test_NewLogsExporter(t *testing.T) {
+
+	type args struct {
+		config *Config
+		logger *zap.Logger
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			"Create NewLogExporter",
+			args{
+				config: &Config{
+					ExporterSettings: config.NewExporterSettings(component.NewID("logicmonitor")),
+					HTTPClientSettings: confighttp.HTTPClientSettings{
+						Endpoint: "http://example.logicmonitor.com/rest",
+					},
+					APIToken: APIToken{AccessID: "testid", AccessKey: "testkey"},
+				},
+				logger: zap.NewNop(),
```

Comment:
I would suggest using `zaptest.NewLogger(t)` so that if it does fail you have some logs to help follow up on.